charts/umbrella/values.yaml

```diff
@@ -121,11 +124,13 @@ dapsserver:
   persistence:
     accessMode: "ReadWriteOnce"
   omejdn:
-    defaultAdminUser: "admin:ThisiSaSuPERSecretPassword1!11" # unknown
+    defaultAdminUser: "admin:ThisiSaSuPERSecretPassword1!11"
```

Comment:
Do not provide hard coded Passwords, even if it's obvious that it is a placeholder. Instead, add a comment to that values explaining how to provide credentials here.

Reply:
@carslen see commit 4771166 - does this fit your requirements?